src/pages/Offer/sections/GetInsured.tsx

```diff
@@ -83,7 +130,7 @@ export const GetInsured: React.SFC<Props> = ({
             {() => (
               <TranslationsConsumer textKey="OFFER_SUMMARY_SIGN_CTA">
                 {(ctaText) => (
-                  <GetInsuredButton>
+                  <GetInsuredButton style={{ marginTop: '30px' }}>
```

Comment:
inline styling :/

Reply:
I know, since the button is global:/ Any nicer way to do that?

Reply:
you can pass props to the button component and pick it up in the styled component :) like this https://github.com/HedvigInsurance/web-onboarding/blob/master/src/components/buttons.tsx#L21